cleanlab/multilabel_classification/filter.py

```diff
@@ -0,0 +1,228 @@
+      columns must be ordered such that these probabilities correspond to
+      class 0, 1, ..., K-1. They need not sum to 1.0
+
+
+    return_indices_ranked_by : {None, 'self_confidence', 'normalized_margin', 'confidence_weighted_entropy'}, default=None
+      Refer to documentation for this argument in :py:func:`filter.find_label_issues <cleanlab.filter.find_label_issues>` for details.
+
+    rank_by_kwargs : dict, optional
+      Refer to documentation for this argument in :py:func:`filter.find_label_issues <cleanlab.filter.find_label_issues>` for details.
+
+    filter_by : {'prune_by_class', 'prune_by_noise_rate', 'both', 'confident_learning', 'predicted_neq_given',
+        'low_normalized_margin', 'low_self_confidence'}, default='prune_by_noise_rate'
+      Refer to documentation for this argument in :py:func:`filter.find_label_issues <cleanlab.filter.find_label_issues>` for details.
+
+    frac_noise : float, default=1.0
+      Refer to documentation for this argument in :py:func:`filter.find_label_issues <cleanlab.filter.find_label_issues>` for details.
+
+    num_to_remove_per_class : array_like
+      Refer to documentation for this argument in :py:func:`filter.find_label_issues <cleanlab.filter.find_label_issues>` for details.
+
+    min_examples_per_class : int, default=1
+      Refer to documentation for this argument in :py:func:`filter.find_label_issues <cleanlab.filter.find_label_issues>` for details.
+
+    confident_joint : np.ndarray, optional
+      An array of shape ``(K, 2, 2)`` representing a one-vs-rest formatted confident joint.
+      Entry ``(c, i, j)`` in this array is the number of examples confidently counted into a ``(class c, noisy label=i, true label=j)`` bin,
+      where `i, j` are either 0 or 1 to denote whether this example belongs to class `c` or not
+      (recall examples can belong to multiple classes in multi-label classification).
+      The `confident_joint` can be computed using :py:func:`count.compute_confident_joint <cleanlab.count.compute_confident_joint>`.
+      If not provided, it is computed from the given (noisy) `labels` and `pred_probs`.
+
+    n_jobs : optional
+      Refer to documentation for this argument in :py:func:`filter.find_label_issues <cleanlab.filter.find_label_issues>` for details.
+
+    verbose : optional
+      If ``True``, prints when multiprocessing happens.
+
+    Returns
+    -------
+    per_class_label_issues : list(np.ndarray)
+      If `return_indices_ranked_by` left unspecified, returns a list of boolean **masks** for the entire dataset
+      where ``True`` represents a label issue and ``False`` represents an example that is
+      accurately labeled with high confidence.
+      If `return_indices_ranked_by` is specified, returns a list of shorter arrays of **indices** of examples identified to have
+      label issues (i.e. those indices where the mask would be ``True``), sorting by likelihood that the corresponding label is correct is not supported yet.
```

Comment:
what does this mean: sorting by likelihood that the corresponding label is correct is not supported yet.

Reply:
Changed here: https://github.com/cleanlab/cleanlab/blob/25672dc504471dd2d450642942555e44528565dc/cleanlab/multilabel_classification/filter.py#L92